packages/sanity/src/_internal/cli/server/getStudioEnvironmentVariables.ts

```diff
@@ -4,29 +4,46 @@ import {loadEnv} from '@sanity/cli'
 const envPrefix = 'SANITY_STUDIO_'
 
 /**
- * Get environment variables prefixed with SANITY_STUDIO_, as an object.
- *
- * Specify `options.prefix` to add a prefix to the environment variable keys,
- * eg: `getStudioEnvironmentVariables({prefix: 'process.env.'})`
- *
- * Specify `options.envFile` to include environment variables from dotenv files (`.env`),
- * in the same way the studio does. A `mode` must be specified, usually `development`
- * or `production`, which will load the corresponding `.env.development` or `.env.production`.
- * To specify where to look for the dotenv files, specify `options.envFile.envDir`.
+ * The params for the `getStudioEnvironmentVariables` function that gets Studio focused environment variables.
  *
- * Specify `options.jsonEncode` to JSON-encode the values, which is handy if you want to pass
- * this to a bundlers hardcoded defines, such as Vite's `define` or Webpack's `DefinePlugin`.
+ * @public
+ */
+export interface studioEnvVariablesOptions {
+  /**
+   * When specified adds a prefix to the environment variable keys,
+   * eg: `getStudioEnvironmentVariables({prefix: 'process.env.'})`
+   */
+  prefix?: string
+  /**
+   * When specified includes environment variables from dotenv files (`.env`), in the same way the studio does.
+   * A `mode` must be specified, usually `development`
+   * or `production`, which will load the corresponding `.env.development` or `.env.production`.
+   * To specify where to look for the dotenv files, specify `options.envFile.envDir`.
+   */
+  envFile?: {mode: string; envDir?: string} | false
+  /**
+   * When specified, JSON-encodes the values, which is handy if you want to pass
+   * this to a bundlers hardcoded defines, such as Vite's `define` or Webpack's `DefinePlugin`.
+   */
+  jsonEncode?: boolean
+}
+
+/**
+ * Get environment variables prefixed with SANITY_STUDIO_, as an object.
  *
  * @param options - Options for the environment variable loading
+ *  {@link studioEnvVariablesOptions}
```

Comment:
Do we need this? I thought TSDoc was smart enough to know which type it was, and thus automatically link it without a specific mention of the type?

Reply:
It does not automatically link to it. The `@link` tag would be needed for anything that we want to link it to.

Reply:
Also updating the link when the interface is changed 
```suggestion
 *  {@link StudioEnvVariablesOptions}
```